src/index.js

```diff
@@ -69,7 +71,7 @@ async function run() {
     });
     logIt({
         message: `checking again in ${moment()
-            .add(FIFTEEN_MINS_MS, 'milliseconds')
```

Comment:
Can you follow the example of how we import `CURRENCY`? It is possible for `TIME_INTERVAL` to be an empty string. Which would break logit && the setInterval below if not set. should be some practical flow typing skill training. Plus the wercker build is failing for the PR because of this.

Visual Studio Code has a really nice flow type plugin that will do something like eslint but for flow.

`npm run flow` will also give you console output for what is going on with the flow state.

Strict typing can be a little 'overkill' sometimes, but it makes everything in your application very explicit and builds the habit of knowing where everything can break and being able to proactively provide better support for such cases.

Perhaps you can even do a "default parameter" for the time interval so that if it was not set up properly it would fall back to the `FIFTEEN_MINS_MS`?